examples/gnrc_border_router/Makefile.native.conf

```diff
@@ -4,9 +4,11 @@ ZEP_PORT_BASE ?= 17754
 ZEP_PORT_MAX := $(shell expr $(ZEP_PORT_BASE) + $(ZEP_DEVICES) - 1)
 
 CFLAGS += -DSOCKET_ZEP_MAX=$(ZEP_DEVICES)
-CFLAGS += -DDHCPV6_CLIENT_PFX_LEASE_MAX=$(ZEP_DEVICES)
 CFLAGS += -DASYNC_READ_NUMOF=$(shell expr $(ZEP_DEVICES) + 1)
 
+# Set CFLAGS if not being set via Kconfig
+CFLAGS += $(if $(CONFIG_KCONFIG_MODULE_DHCPV6),,-DCONFIG_DHCPV6_CLIENT_PFX_LEASE_MAX=$(ZEP_DEVICES))
```

Comment:
@miri64 is this configuration being used? I can't see `dhcpv6` in the list of modules when compiling for native. Is this for some special case?

Reply:
Did you compile with `USE_DHCPV6=1` set in the environment?

Reply:
@miri64  If I am to add. I used `USEMODULE += DHCPV6` in the [Makefile.native.conf](https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_border_router/Makefile.native.conf) and enabled Kconfig by commenting this [line](https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_border_router/Makefile#L106).

I just used `USE_DHCPV6=1` instead of ` USEMODULE += DHCPV6 ` and the option is available via meniconfig

Reply:
> Did you compile with `USE_DHCPV6=1` set in the environment?

There it is. Thanks!